libs/backend/src/cast_vote_records/export.ts

```diff
@@ -72,12 +71,36 @@ export interface ScannerStore {
     castVoteRecordId: string,
     castVoteRecordHash: string
   ): void;
+}
 
-  getExportDirectoryName?(): string | undefined;
-  getPollsState?(): PollsState;
-  setExportDirectoryName?(exportDirectoryName: string): void;
+/**
+ * The subset of central scanner store methods relevant to exporting cast vote records
+ */
+export interface CentralScannerStore extends ScannerStoreBase {
+  scannerType: 'central';
 }
 
+/**
+ * The subset of precinct scanner store methods relevant to exporting cast vote records
+ */
+export interface PrecinctScannerStore extends ScannerStoreBase {
+  scannerType: 'precinct';
+
+  getBallotsCounted(): number;
+  getExportDirectoryName(): string | undefined;
+  getPollsState(): PollsState;
+  isContinuousExportOperationInProgress(): boolean;
+  setExportDirectoryName(exportDirectoryName: string): void;
+  setIsContinuousExportOperationInProgress(
+    isContinuousExportOperationInProgress: boolean
+  ): void;
```

Comment:
Rather than having all these methods be optional on the `ScannerStore` interface, I've now updated `ScannerStore` to be a union of two distinct interfaces, one for the central scanner and one for the precinct scanner (still sharing what I can though, via `ScannerStoreBase`)